src/lists.h

```diff
@@ -592,7 +592,7 @@ EXPORT_INLINE Int IS_TABLE_LIST(Obj list)
 *F  IS_SSORT_LIST(<list>) . . . . . . . . . .  test for strictly sorted lists
 *V  IsSSortListFuncs[<type>]  .  table of strictly sorted list test functions
 **
-**  'IS_SSORT_LIST' returns 2 if the list <list> is  a strictly  sorted  list
+**  'IS_SSORT_LIST' returns 1 if the list <list> is  a strictly  sorted  list
```

Comment:
Note that I changed this to always return 0 or 1. Many (most?) of the functions installed in `IsSSortListFuncs` already return 0 or 1, but a few returned 0 or 2 as documented. Luckily, all callers (there are not many) only care whether the return value is zero or non-zero, so this didn't matter.

By changing the return value to `BOOL` and using `return TRUE` and `return FALSE`, I think this code got clearer.